travis.sh

```diff
@@ -40,7 +40,7 @@ if [ -n "${FORMAT:-}" ]; then
     fi
     git clang-format-3.8 --binary=/usr/bin/clang-format-3.8 --style=file --commit="${BASE_HEAD}"
     # Assert that all changes adhere to the asked for style
-    exec git diff --exit-code
+    exec git diff --exit-code -- . ":!3rdparty"
```

Comment:
This change doesn't belong to b22cbf71a77625200d543c21a99c9d791e6e8bc8 (_Recalculate position so it is based on the index of unicode codepoints_). It should be in a commit of its own, or at least in the commit that updates `json-spirit`.